pegasus-spark-analyser/src/main/java/com/xiaomi/infra/pegasus/spark/analyser/PegasusScanner.java

```diff
@@ -0,0 +1,14 @@
+package com.xiaomi.infra.pegasus.spark.analyser;
+
+public interface PegasusScanner {
```

Comment:
It shouldn't be a public interface. The user must always use the RDD abstraction provided by pegasus-analyzer. `PegasusScanner` is hidden under RDD.

Reply:
java `Interface` default public and need public which meas can be inherited and overload. Of course, you can put one interface into another class to hide it. But I think it's not necessary.